multimedia/gst1-libav/Makefile

```diff
@@ -182,11 +197,7 @@ CONFIGURE_ARGS += \
 	--disable-fma4 \
 	--disable-avx2 \
 	--disable-inline-asm \
-	--disable-yasm"
-
-TARGET_CFLAGS += -D_GNU_SOURCE -Wno-format-nonliteral
-
-TARGET_LDFLAGS += $(FPIC)
+	--disable-x86asm"
```

Comment:
@neheb @MikePetullo 
Do we really need to use the sledgehammer approach and kill all kind of performance?
I know it can get a bit kludgy (https://github.com/diizzyy/packages/blob/ffmpeg4/multimedia/ffmpeg4/Makefile#L202) but I think we should at least try to optimize ARM and possibly x86.

Reply:
I switched it to x86asm as the configure script indicated --enable-yasm is deprecated and that --enable-x86asm should be used. My guess is yasm is used for x86 because of some issues with GAS (GNU Assembler), which is what is used with other platforms.